.github/workflows/R-CMD-check.yaml

```diff
@@ -44,16 +43,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set locale
-        if: matrix.config.locale == 'en_US'
-        run: |
-          sudo locale-gen en_US
-          echo "LC_ALL=en_US" >> $GITHUB_ENV
-
-      - name: Install XQuartz on macOS
-        if: runner.os == 'macOS'
-        run: brew install --cask xquartz
-
```

Comment:
Off the top of my head, I don't see why this is necessary here in vroom. Can you look at git blame / git history to see if you can discover any reason we (still) need this?

Reply:
Here is where it was originally added

https://github.com/r-lib/vroom/commit/08e483877cfa0a87c45fcb739892c319d349b6aa

But more recently, some similar code from `.github/workflows/pkgdown.yaml` was removed here:

https://github.com/r-lib/vroom/commit/783c3fa5d082e823cf7d586d82f0c56fa0fd125e

Which I think supports that we can remove this?

Reply:
I think it's safe to remove. We also saw CI pass (except for the job that is failing due to the rmarkdown problem) with this deleted, in this PR.